internal/renderers/skia/textlayout.rs

```diff
@@ -99,13 +99,9 @@ pub fn create_layout(
 
     let mut style = skia_safe::textlayout::ParagraphStyle::new();
 
-    if overflow == items::TextOverflow::Elide {
+    if overflow == items::TextOverflow::Elide && wrap == items::TextWrap::NoWrap {
         style.set_ellipsis("…");
-        if wrap != items::TextWrap::NoWrap {
-            let metrics = text_style.font_metrics();
-            let line_height = metrics.descent - metrics.ascent + metrics.leading;
-            style.set_max_lines((max_height.get() / line_height).floor() as usize);
```

Comment:
What was wrong with this computation? Trying to avoid an extra layout seems worth it